src/Symfony/Bridge/PhpUnit/bin/simple-phpunit

```diff
@@ -230,9 +230,8 @@ if ($components) {
     if (!class_exists('SymfonyBlacklistSimplePhpunit', false)) {
         class SymfonyBlacklistSimplePhpunit {}
     }
-    array_splice($argv, 1, 0, array('--colors=always'));
-    $_SERVER['argv'] = $argv;
-    $_SERVER['argc'] = ++$argc;
```

Comment:
this line should not be removed!

Reply:
I forgot to put them back while I was debugging the cli and phpstorm problems.

Reply:
I forgot to put them back while I was debugging. They are back in 67cc253.